modules/tinymce/src/plugins/visualchars/test/ts/atomic/DataTest.ts

```diff
@@ -1,22 +1,18 @@
-import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+
 import * as Data from 'tinymce/plugins/visualchars/core/Data';
 
-UnitTest.test('atomic.tinymce.plugins.visualchars.DataTest', () => {
-  Assert.eq(
-    'should return correct selector',
-    'span.mce-a,span.mce-b',
-    Data.charMapToSelector({ a: 'a', b: 'b' })
-  );
+describe('atomic.tinymce.plugins.visualchars.DataTest', () => {
+  it('should return correct selector', () => {
```

Comment:
Does it make sense to break this up into 3 different tests? It wasn't what I was expecting, as I would have just made it 2 tests for each function with 3 assertions. eg:
```
it('charMapToSelector', () => {
  assert.equal('span.mce-a,span.mce-b', Data.charMapToSelector({ a: 'a', b: 'b' }), 'should return correct selector');
});

it('charMapToRegExp', () => {
  assert.equal('/[ab]/', Data.charMapToRegExp({ a: 'a', b: 'b' }).toString(), 'should return correct regexp');
  assert.equal('/[ab]/g', Data.charMapToRegExp({ a: 'a', b: 'b' }, true).toString(), 'should return correct global regexp');
});
```

Reply:
As discussed, we're going to keep this style. However, this does need to be wrapped in a context so it's clear what part of `DataTest` it's testing.

- [ ] Wrap in context for each function being tested